orchagent/fdborch.cpp

```diff
@@ -46,7 +47,7 @@ bool FdbOrch::getPort(const MacAddress& mac, uint16_t vlan, Port& port)
     entry.vlan_id = vlan;
 
     sai_attribute_t attr;
-    attr.id = SAI_FDB_ENTRY_ATTR_PORT_ID;
+    attr.id = SAI_FDB_ENTRY_ATTR_BRIDGE_PORT_ID;
```

Comment:
Just make sure that you will correctly get a "class Port" instance by "bridge port id".